src/helpers/commonAction/checkExistance.js

```diff
@@ -0,0 +1,38 @@
+import statusCode from '../constants/status.codes';
+import sendError from '../error.sender';
+/* istanbul ignore next: */
+export default async (req, res, next, model, item) => {
+  /**
+   * @author Daniel Mwangila
+   * @description Object.entries gets all the entries
+   * that are in the req.params and
+   * put them into a two dimensional array
+   * e.g [[slug, How-to-seed]]
```

Comment:
@truestbyheart if this file only checks for an article's existence, you might want to change it to `checkArticleExistence`

Reply:
@MemunaHaruna  this function checks for everything existence scenario just give it a parameter to check and it will do it...  I created this one especially for Articles and comments
it just extracts the param value and generates a ``where`` object to query the database.
the description is to explain the inner working of it